ci/jenkins/test.sh

```diff
@@ -734,9 +749,22 @@ function run_conformance {
     kubectl rollout status daemonset/antrea-agent -n kube-system
 
     set +e
+    # Conformance dns test cases fail in ipv6 kind cluster because here CoreDNS pods are ipv6 pods and 
+    # it can't reach ipv4 upstream namespace server.
+    # Note: By default CoreDNS ConfigMap uses /etc/resolv.conf to get upstream nameservers which can be accessed by ipv4 and 
+    # single stack ipv6 CoreDNS would not be able to connect that server and it return servfail, 
```

Comment:
Can you reorg this sentence to make it clear? Do you mean the upstream nameservers can be accessed by ipv4, but single stack ipv6 CoreDNS would not be able to connect that server and it return servfail?